core-test/src/test/scala/org/bitcoins/core/crypto/SchnorrTest.scala

```diff
@@ -0,0 +1,166 @@
+package org.bitcoins.core.crypto
+
+import org.bitcoin.NativeSecp256k1
+import org.bitcoins.core.util.{CryptoUtil, NumberUtil}
+import org.bitcoins.testkit.core.gen.{CryptoGenerators, NumberGenerator}
+import org.bitcoins.testkit.util.BitcoinSUnitTest
+import scodec.bits.ByteVector
+
+class SchnorrTest extends BitcoinSUnitTest {
+  implicit override val generatorDrivenConfig: PropertyCheckConfiguration =
+    generatorDrivenConfigNewCode
+
+  behavior of "NativeSecp256k1"
+
+  it must "act like a normal schnorrSign if the normal nonce is specified" in {
+    forAll(CryptoGenerators.nonZeroPrivKey, NumberGenerator.bytevector(32)) {
+      case (privKey, message) =>
+        val nonce = SchnorrNonce.fromBipSchnorr(privKey, message)
```

Comment:
comment saying 

//first, let's manually generate the nonce specified in bip-schnorr inside of bitcoin-s and then pass it into secp256k1 and get a sig back